docs/tutorials/BuildMultiClusterApplication/BuildMultiClusterApplication.md

```diff
@@ -63,19 +65,21 @@ spec:
     overrideRules:
       - clusterSelector:
           clusterSelectorTerms:
-            - labelSelector:
-                matchLabels:
-                  cluster-name: member-1
+            - labelSelector: {} # select all the clusters
         jsonPatchOverrides:
           - op: add
             path: /metadata/annotations
             value:
-              {"service.beta.kubernetes.io/azure-dns-label-name":"fleet-multi-cluster-app-member-1"}
+              {"service.beta.kubernetes.io/azure-dns-label-name":"multi-cluster-app-${MEMBER-CLUSTER-NAME}"}
 ```
 Summary:
 - This defines a Kubernetes Service named `nginx-service` in the `test-app` namespace.
 - The service is of type LoadBalancer with a public ip address and a DNS name assigned.
 - It targets pods with the label app: nginx and forwards traffic to port 80 on the pods.
+- The override rule assigns a DNS name to the service based on the cluster name.
+
+> Note: Please update the dns label name to match your specific requirements, and the "multi-cluster-app-${MEMBER-CLUSTER-NAME}" may be not available.
+
 
 #### Deployment

```

Comment:
The namespace of the deployment inside the envelope-configmap still uses test-app instead of multi-cluster-app